DomainModels/IModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DomainModels
+{
+    public interface IModel
+    {
+        int Id { get; set; }
+    }
```

Comment:
Utilizar long para Id ao invés de int.
```suggestion
        long Id { get; set; }
    }
```